web/css/events.css

```diff
@@ -91,6 +91,12 @@
   color: #666;
 }
 
+.map-item-list li.item p.magnitude {
```

Comment:
Another nitpick but, for styles that share selectors I think it's preferable to nest them to keep them logically grouped in the stylesheet and also to reduce redundant definitions both of which improve readability.  Since this selector and the one below both share `.map-item-list li.item p` you could nest the rest like so:

```
.map-item-list li.item p {
   &.magnitude {
     // these styles
   }
   a {
      // styles from the block below
   }
}
```

A lot of our older CSS does not follow this convention but I have been slowly trying to migrate as much of it as I can when working within a file.